lib/matplotlib/cbook.py

```diff
@@ -2327,3 +2327,95 @@ def get_instancemethod(self):
 else:
     def _putmask(a, mask, values):
         return np.copyto(a, values, where=mask)
+
+class OrderedDict(dict):
```

Comment:
I agree, just one question, why did you implemented by yourself instead of using the "recommended" recipe `http://code.activestate.com/recipes/576693/`